ReSharper.FSharp/src/FSharp.Psi.Features/src/CodeCompletion/Rules/FSharpRelevanceRule.fs

```diff
@@ -39,6 +41,18 @@ type FSharpRelevanceRule() =
                 if mfv.IsMember && mfv.IsProperty then
                     markRelevance item CLRLookupItemRelevance.FieldsAndProperties
                 else
+                    if fcsLookupItem.FcsSymbolUse.IsFromComputationExpression then
+                        let isCustomOperationPossible =
+                            let reference = context.ReparsedContext.Reference
+                            if isNull reference then false else
+
+                            let refExpr = reference.GetTreeNode().As<IReferenceExpr>()
+                            if isNull refExpr then false else
+
+                            isInsideComputationExpressionForCustomOperation refExpr
+                        if isCustomOperationPossible then
+                            markRelevance item CLRLookupItemRelevance.ExpectedTypeMatch
```

Comment:
`ExpectedTypeMatch` is used only because it is top relevance. Maybe we want add new relevance?